### ISSUE TITLE
Add 'Search Queries' support

### ISSUE THREAD

**Issue by thomas-peyric** [liquidm/ruby-druid#29]
Add support of 'Search Queries' which returns dimension values that match the search specification ( http://druid.io/docs/latest/SearchQuery.html )

**Comment by hollow** [comment]
can this be closed?

**Comment by itszootime** [comment]
Yep - it's been implemented (with different syntax) in the ruby-druid team fork.
